src/services/facilityService.ts

```diff
@@ -1,21 +1,24 @@
 import { Facility } from '../typeDefs/gqlTypes'
-import {Facility as PrismaFacility, Contact as PrismaContact } from '@prisma/client'
+import {Facility as PrismaFacility, 
+    Contact as PrismaContact, PhysicalAddress as PrismaPhysicalAddress } from '@prisma/client'
 
 import prisma from '../db/client'
 
 type FacilityAndRelations = (PrismaFacility & {
-    contact: PrismaContact;
+    contact: PrismaContact,
+    physicalAddress: PrismaPhysicalAddress
 })
 
 function convertPrismaToGraphQLFacility(input: FacilityAndRelations | null) {
     if (!input) { return null }
     return {
         id: String(input.id),
-        nameJa: input.nameJa,
         nameEn: input.nameEn,
+        nameJa: input.nameJa,
         contact: input.contact,
+        physicalAddress: input.physicalAddress,
         healthcareProfessionals: []
-    } as Facility
+    } as unknown as Facility
```

Comment:
Could use some input on how to handle this better. Without the `unknown` I get an error about the physicalAddress.id type not matching.